csrc/scheduler/normalization.cpp

```diff
@@ -516,18 +516,18 @@ std::shared_ptr<ReductionParams> innerPersistentHeuristic(
   const int64_t threads_after_vectorize =
       inner_most_dimension_numel / inner_reduction_unroll_factor;
   constexpr int64_t scheduler_per_sm = 4;
+  const int64_t min_threads_per_block = scheduler_per_sm * dev_prop->warpSize;
```

Comment:
Shouldn't this be `min_threads_per_sm`?

Reply:
It should be `min_threads_per_block` because we want to use it to decide whether we need to do multi reductions per block. I know here `scheduler_per_sm` is confusing.

Reply:
Hmm, there are `scheduler_per_sm` schedulers on each SM, and we want to feed each of them at least 32 threads, so it sounds like the minimum number of threads we would want to have on each SM.

The size of each thread block doesn't need to be as large as `scheduler_per_sm * dev_prop->waprSize` to fill all the schedulers, right?